src/components/Form/Radiobutton.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { Component, ReactNode } from 'react';
+import classnames from 'classnames';
+import { HtmlInput, HtmlLabel, HtmlSpan, HtmlDiv } from '../../reset';
+import { idGenerator } from '../../utils/uuid';
+import { logger } from '../../utils/logger';
+import {
+  RadiobuttonGroupConsumer,
+  RadiobuttonGroupProviderState,
+} from './RadiobuttonGroup';
+
+const baseClassName = 'fi-radiobutton';
+
+const radiobuttonClassNames = {
+  container: `${baseClassName}_container`,
+  input: `${baseClassName}_input`,
+  label: `${baseClassName}_label`,
+  hintText: `${baseClassName}_hintText`,
+};
+
+type RadiobuttonVariant = 'small' | 'large';
+
+export interface RadiobuttonProps {
```

Comment:
No need for common input props?

Reply:
Not sure. I used  **Checkbox** as an example to look guidance and to keep it similar.

If trying to use `extends Omit<HtmlInputProps, 'type'>` on the Radiobutton it complains of multiple problems, as currently the `...passProps` are passed to **HtmlLabel** as with the Checkbox.

But I'm not sure if we want **Radiobutton** and **Checkbox** to give those `...passProps` to **HtmlLabel** or **HtmlInput**. Do you remember @LJKaski was there some special reason to pass them to HtmlLabel?

Reply:
That is actually a valid point, we do have additional stuff in addition to that actual input. I still think that passprops should go to the input ja there might be quite a few common input props that we may need to support later on unless support is added now. Omitting type from common props should do the trick, but there might be too much stuff in the common props, so not sure what would be the best approach.

Reply:
I think that might just be an error in Checkbox as well. Passing to the input makes more sense for sure. In general I think it is easier to just pass the input props and omit the ones with the most obvious clashing potential.

Reply:
After digging through HtmlInputProps which is essentially HtmlInputElement from lib.dom.d.ts, it looks like there are way too many native attributes causing or enabling unwanted effects e.g. with styling. Considering this it was agreed to not add common input props.